base.docker

```diff
@@ -3,6 +3,7 @@ FROM python:3.6.1-slim
 ENV APP_DIR /app
 
 ENV NODE_VERSION 6.10.3
```

Comment:
Is there a reason we're still leaving node in here?

Reply:
I think we still need node on our apps to build our assets, yarn is just replacing bower and npm